DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -732,54 +730,25 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
         try await tunnelManager.saveToPreferences()
     }
 
-    /* Temporarily disabled until we fix this menu: https://app.asana.com/0/0/1205766100762904/f
```

Comment:
Removing a ton of old code we'll never use again.

Reply:
Nice 👏